### PR TITLE
Feature panel

### PR DESCRIPTION
**Pull request by dhaval2989**
Feature panel

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -21,7 +21,8 @@ import { ReusableModule } from './reusable/reusable.module';
     AppRoutingModule,
     BrowserAnimationsModule,
     MaterialModule,
-    LayoutComponentModule
+    LayoutComponentModule,
+    ReusableModule
```
Comment (veeranjaneyuludavuluri) [inline src/app/app.module.ts:25]: are we using layout components and reusable components in app module?

File: src/app/reusable/panel/panel.component.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+          Panel Title
+          <button type="button" class="close" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="card-body ">
+            <div class="shadow-div shadow-sm">
+
+            </div>
+          <!-- <h5 class="card-title">Special title treatment</h5>
```
Comment (Sina-pz) [inline src/app/reusable/panel/panel.component.html:13]: Please delete the comments

**Review by Sina-pz — APPROVED** [review]

File: src/app/auth/auth.module.ts [hunk 3]
```diff
@@ -4,6 +4,7 @@ import { LoginComponent } from './login/login.component';
 import { MaterialModule } from '../material.module';
 import { ReusableModule } from '../reusable/reusable.module';
 import { RouterModule } from '@angular/router';
+import { MainRoutingModule } from '../main/main-routing.module';
```
Comment (veeranjaneyuludavuluri) [inline src/app/auth/auth.module.ts:7]: why are you using main routing module in auth module?

File: src/app/app.module.ts (hunk 1 above)
Comment (dhaval2989) [reply src/app/app.module.ts:25]: yeah, its depend upon functionality.

File: src/app/reusable/panel/panel.component.html (hunk 2 above)
Comment (dhaval2989) [reply src/app/reusable/panel/panel.component.html]: resolved

File: src/app/app.module.ts (hunk 1 above)
Comment (veeranjaneyuludavuluri) [reply src/app/app.module.ts]: we are using lazy loading and no components are used in app module, if we import other child modules into the app module then the bundle file will increase, and using lazy loading is useless.
Even you are not using layout module components in app module, you were using them in the main module right then why don't you import them in main?

Comment (dhaval2989) [reply src/app/app.module.ts]: resolved

File: src/app/auth/auth.module.ts (hunk 3 above)
Comment (dhaval2989) [reply src/app/auth/auth.module.ts]: resolved

File: src/app/app.module.ts [hunk 4]
```diff
@@ -17,6 +18,7 @@ import { LayoutComponentModule } from './layout/layout.module';
     BrowserAnimationsModule,
     MaterialModule,
     LayoutComponentModule,
```
Comment (veeranjaneyuludavuluri) [inline src/app/app.module.ts:20]: I hope we are not using any components of layout module in app module then why we are importing here

Comment (dhaval2989) [reply src/app/app.module.ts]: resolved

File: src/app/app.module.ts (hunk 1 above)
Comment (dhaval2989) [reply src/app/app.module.ts]: update it

**Review by veeranjaneyuludavuluri — APPROVED** [review]
Good to go

**Comment by veeranjaneyuludavuluri** [comment]
please resolve the conflicts and merge into development

**Review by dhaval2989 — COMMENTED** [review]
resolve commit done

**Pull request merged**